README.md

```diff
@@ -4,9 +4,9 @@
 
 # Microsoft SqlClient Data Provider for SQL Server
 
-Welcome to the home of Microsoft ADO.NET driver for SQL Server aka the Microsoft.Data.SqlClient GitHub repository.
+Welcome to the home of the Microsoft ADO.NET driver for SQL Server aka the Microsoft.Data.SqlClient GitHub repository.
 
-Microsoft.Data.SqlClient is a data provider for Microsoft Sql Server. Now in General Availability, it is a union of the two System.Data.SqlClient components which live independently in .NET Framework and .NET Core. Going forward, support for new SQL Server features will be implemented in Microsoft.Data.SqlClient.
+Microsoft.Data.SqlClient is a data provider for Microsoft SQL Server and Azure SQL Database. Now in General Availability, it is a union of the two System.Data.SqlClient components which live independently in .NET Framework and .NET Core. Going forward, support for new SQL Server features will be implemented in Microsoft.Data.SqlClient.

```

Comment:
Considering this driver will be documented here in "ADO.NET" section soon: https://docs.microsoft.com/en-us/sql/connect/sql-connection-libraries?view=sql-server-ver15 with other language providers, all drivers are defined as "Microsoft <> driver for SQL Server". In descriptions/details there's always mention of "Azure" databases.

Wondering if this change is needed in this manner. Or could we add another sentence to explain its usage as _"interacting with Microsoft SQL Server, and with its twin in the cloud Azure SQL Database."_ ?

Reply:
Maybe: Microsoft.Data.SqlClient is a data provider for Microsoft SQL Server and its twin in the cloud Azure SQL Database.

Reply:
I think the change is appropriate. Azure SQL DB is growing in usage and focus. I would not refer to it as "its twin in the cloud". Marketing would probably cringe.

Reply:
@David-Engel Agree, I guess this page needs an overhaul: https://docs.microsoft.com/en-us/sql/connect/sql-connection-libraries?view=sql-server-ver15

Reply:
Yes, it does. I'll put it on my to-do list.